types/encoding.go

```diff
@@ -0,0 +1,16 @@
+package types
+
+import (
+	"github.com/cosmos/cosmos-sdk/client"
+	"github.com/cosmos/cosmos-sdk/codec"
+	"github.com/cosmos/cosmos-sdk/codec/types"
+)
+
+// EncodingConfig specifies the concrete encoding types to use for a given app.
```

Comment:
Yeah this is needed because it basically just copies the structure out of simapp. You can see the `ethermint.EncodingConfig`.

It's imported in `encoding/config.go`